buildSrc/src/main/groovy/com.logicmonitor.java-conventions.gradle

```diff
@@ -16,7 +16,7 @@ dependencies {
 }
 
 group = 'com.logicmonitor'
-version = '0.0.4-alpha'
+version = '0.0.4-alpha-govCloud'
```

Comment:
We can increase the version instead of appending anything new. The same SDK will also be available for regular accounts.